src/jobstats.rs

```diff
@@ -170,110 +170,156 @@ type JobStatMdtPromInst<'a> = (
     PrometheusInstance<'a, i64, Yes>,
     PrometheusInstance<'a, i64, Yes>,
     PrometheusInstance<'a, i64, Yes>,
+    Option<PrometheusInstance<'a, i64, Yes>>,
+    Option<PrometheusInstance<'a, i64, Yes>>,
 );
 
 fn jobstatmdt_inst<'a>(
     x: &'a JobStatMdt,
     kind: &'a str,
     target: &'a str,
 ) -> JobStatMdtPromInst<'a> {
+    let JobStatMdt {
+        job_id,
+        snapshot_time: _,
+        start_time: _,
+        elapsed_time: _,
+        open,
+        close,
+        mknod,
+        link,
+        unlink,
+        mkdir,
+        rmdir,
+        rename,
+        getattr,
+        setattr,
+        getxattr,
+        setxattr,
+        statfs,
+        sync,
+        samedir_rename,
+        crossdir_rename,
+        read_bytes: _,
+        write_bytes: _,
+        punch: _,
```

Comment:
```suggestion
        read_bytes: _,
        write_bytes: _,
        punch: _,
```

While we are there, I wonder if we should include those remaining stats. Might be useful to know how often DoM is used